pandas/tests/extension/base/getitem.py

```diff
@@ -388,7 +388,8 @@ def test_loc_len1(self, data):
         # see GH-27785 take_nd with indexer of len 1 resulting in wrong ndim
         df = pd.DataFrame({"A": data})
         res = df.loc[[0], "A"]
-        assert res._mgr._block.ndim == 1
+        if hasattr(res._mgr, "blocks"):
+            assert res._mgr._block.ndim == 1
```

Comment:
i think OK to just check res._mgr.ndim (or even just res.ndim)

Reply:
I added a check for the ndim of mgr (so there is also a check for ArrayManager), but left the block ndim check in place, because the original bug was that *only* the block ndim was wrong (the manager ndim was correct)